closed/openssl.gmk

```diff
@@ -63,10 +63,12 @@ endif # windows
 # Identify the desired openssl target configuration.
 OPENSSL_TARGET :=
 ifeq ($(OPENJDK_TARGET_OS), aix)
-  ifeq ($(TOOLCHAIN_TYPE), clang)
-    OPENSSL_TARGET := aix64-clang
-  else ifeq ($(TOOLCHAIN_TYPE), xlc)
-    OPENSSL_TARGET := aix64-cc
+  ifeq ($(OPENJDK_TARGET_CPU), ppc64)
+    ifeq ($(TOOLCHAIN_TYPE), clang)
+      OPENSSL_TARGET := aix64-clang
+    else ifeq ($(TOOLCHAIN_TYPE), xlc)
+      OPENSSL_TARGET := aix64-cc
+    endif
```

Comment:
I was expecting the misconfiguration would be detected by `configure`, not later during a build (attempt).

Reply:
Worse, if openssl appears to be built (`libcrypto.so` and `libssl.so` are present), it doesn't try to build it, but would use (bundle) an incompatible version.

Reply:
Updated.